src/components/Button/src/Button.vue

```diff
@@ -241,17 +349,19 @@ export default {
 	vertical-align: middle;
 	background-color: var(--color-main, $maker-color-primary);
 	border: none;
-	border-radius: $maker-shape-button-border-radius;
+	border-radius: var(--border-radius, $maker-shape-button-border-radius);
 	outline: none;
 	box-shadow:
-		var(--outline-border, 0 0),
+		var(--border, 0 0),
+		var(--box-shadow, 0 0),
 		var(--focus-border, 0 0);
 	cursor: pointer;
 	transition:
 		color 0.2s ease-in,
 		background-color 0.2s ease-in,
 		filter 0.2s ease-in,
-		box-shadow 0.2s ease-in;
+		box-shadow 0.2s ease-in,
+		border-radius 0.2s ease-in;
```

Comment:
since `border-width` and `border-color` can also be customized should we add those here as well?

Reply:
We don't actually set `border-color` or `border-width`, they're actually styled as box-shadows too.